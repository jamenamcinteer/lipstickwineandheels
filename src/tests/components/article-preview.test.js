import React from 'react'
import { shallow } from 'enzyme'
import ArticlePreview from '../../components/article-preview'
import data from '../fixtures/data'

test('should render article preview component correctly', () => {
  const wrapper = shallow(<ArticlePreview article={data.data.contentfulBlog} />)
  expect(wrapper).toMatchSnapshot()
})
