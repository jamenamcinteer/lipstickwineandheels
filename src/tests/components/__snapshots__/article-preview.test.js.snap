// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render article preview component correctly 1`] = `
<div
  className="preview"
>
  <div
    className="imageContainer"
  >
    <Image
      Tag="div"
      alt="this is the image title"
      fadeIn={true}
    />
  </div>
  <div
    className="contentContainer"
  >
    <h2
      className="previewTitle"
    >
      <GatsbyLink
        to="/blog/this-is-a-sample-blog-post"
      >
        This is a sample blog post
      </GatsbyLink>
    </h2>
    <p
      className="tag"
    >
      <GatsbyLink
        key="tech"
        to="/tag/tech"
      >
        Tech
      </GatsbyLink>
    </p>
    <small>
      August 30, 2018
       
      <span
        className="bullet"
      >
        â€¢
      </span>
      1 min read
    </small>
    <p
      className="articleReadMore"
    >
      <GatsbyLink
        className="button button-secondary"
        to="/blog/this-is-a-sample-blog-post"
      >
        Continue Reading
      </GatsbyLink>
    </p>
  </div>
</div>
`;
